packages/kit/src/node.js

```diff
@@ -47,3 +49,27 @@ export function getRawBody(req) {
 		});
 	});
 }
+
+/** @type {import('@sveltejs/kit/node').GetRequest} */
+export async function getRequest(base, req) {
+	return new Request(base + req.url, {
```

Comment:
should we copy the other request data such as the referrer as well?

Reply:
not sure I understand? we're passing through the headers, is there something else we're missing?

Reply:
I was looking at the other [constructor options on MDN](https://developer.mozilla.org/en-US/docs/Web/API/Request/Request) and wondering if we needed any of them?  `mode`, `credentials`, `cache`, `redirect`, `referrer`, `integrity`